source/release-notes/v2.0-release-notes.rst

```diff
@@ -178,6 +178,13 @@ Details
 Pinned Apps: Enhanced app launch interface using large app icons on the dashboard
 .................................................................................
 
+2.0 now allows sites to pin a grid of application icons to the dashboard for easy access
+and to give prominence to the apps. The grid layout of application icons is is meant to
+give users a desktop look and feel to the dashboard.
+
+There are several strategies available to choose which apps to pin. Sites can pin by type
+and/or by metadata available in the apps manifest file.
```

Comment:
For example, metadata in the app manfests could specify a field_of_science attribute, and then the pinned apps could be configured to display all apps with the field_of_science being "Biology". The configuration for pinned apps can be made dynamic using ERB so it can be changed based on which user or group is accessing the dashboard. Pinned apps can also further be grouped by a particular attribute.

Reply:
>for easy access and to give prominence to the apps

for easy access and to draw attention to a specific subset of apps.


I think that's what you meant, but what was lost in the "prominence to the apps" is the fact that this feature lets us hide most of the apps, and only show a few of interest.

Reply:
All that information from the first comment is in the actual docs. Are you saying we want to put all this information here _and_ there?

Reply:
Yes. I expect the actual docs to go into greater detail.

Reply:
My "for example" paragraph was suggested to be added verbatim to the bottom of this.

Reply:
A possible intro to the section as well:

As the number of apps increases and the sophtication of the typical user decreases - now including even undergraduate students using OnDemand in the classroom - it has become desirable to be able to present only a small subset of the apps that are relevant for a particular user.